configs/coins/decred.json

```diff
@@ -0,0 +1,68 @@
+{
+    "coin": {
+      "name": "Decred",
+      "shortcut": "DCR",
+      "label": "Decred",
+      "alias": "decred"
+    },
+    "ports": {
+      "backend_rpc": 9109,
+      "backend_message_queue": 38888,
+      "blockbook_internal": 10108,
+      "blockbook_public": 11108
```

Comment:
Could you please use ports 8061, 38361, 9061, 9161? And 18061, 48361, 19061, 19161 for the testnet? It will make it easier to manage potential conflicts with other coins. Thank you.